mbed_greentea/mbed_greentea_cli.py

```diff
@@ -696,6 +696,7 @@ def get_parallel_value(value):
 
     # Detect devices connected to system
     mbeds = mbed_lstools.create()
+    mbeds.mbedls_get_mocks()
```

Comment:
greentea should not invoke internal working of mbed-ls like. Perhaps mbed-ls should be changed to include mocks via create. Just as it does on command line.